experimental/PropertyDDS/packages/property-properties/src/properties/baseProperty.ts

```diff
@@ -589,9 +586,8 @@ export abstract class BaseProperty {
 	 * @param {property-properties.PathHelper.TOKEN_TYPES} in_segmentType - The type of segment in the tokenized path
 	 *
 	 * @return {property-properties.BaseProperty|undefined} The child property that has been resolved
-	 * @protected
 	 */
-	_resolvePathSegment(in_segment: string, in_segmentType: PathHelper.TOKEN_TYPES) {
+	protected _resolvePathSegment(in_segment: string, in_segmentType: PathHelper.TOKEN_TYPES) {
```

Comment:
TypeScript ignores @protected JSDoc annotations for *.ts files.  Fixed by using the 'protected' keyword instead.